.github/workflows/cspell.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Codespell
+        run: |
+          npm install -g cspell@latest
```

Comment:
I had to comment it out, it was too noisy to actually run. But by getting many of the files in now (in effectively noop mode) I felt like it will pay dividends in the future by keeping it small.

Reply:
ok, it's working again (just on one file to start)